e2e/pages/events/manageHearingsEvent.page.js

```diff
@@ -29,9 +29,16 @@ module.exports = {
     newVenueCustomAddress: '#previousHearingVenue_newVenueCustomAddress_newVenueCustomAddress',
     startDate: '#hearingStartDate',
     endDate: '#hearingEndDate',
-    sendNotice: '#sendNoticeOfHearing_Yes',
-    dontSendNotice: '#sendNoticeOfHearing_No',
-    noticeNotes: '#noticeOfHearingNotes',
```

Comment:
does this ticket require this to be removed?

Reply:
Looks like they accidentally removed it from the designs, I have added it back